### DISCUSSION TITLE
Transferring browser-previews to nix-community

### DISCUSSION THREAD

**Discussion by JohnRTitor** [nix-community/infra#1247] [Q&A] [answered]
Is [r-k-b/browser-previews](https://github.com/r-k-b/browser-previews) eligible to be hosted at nix-community?

If yes how one should go about this? @r-k-b [showed](https://github.com/r-k-b/browser-previews/issues/8#issuecomment-1937889020) an interest in migrating this over.

There is also a suggestion by @surfaceflinger to merge [nix-community/flake-firefox-nightly](https://github.com/nix-community/flake-firefox-nightly) with this one. The merged repo would host both Chrome and Firefox, stable and nightlies.

CC: @r-k-b @surfaceflinger @colemickens @Artturin

**Comment by zowoq** [comment]
@r-k-b I've sent you an invite to join nix-community and created a team @nix-community/browser-previews.

**Reply by r-k-b** [reply, marked as answer]
Thanks! Seems like the old urls will continue to work, so I've transferred the repo just now.
